### DISCUSSION TITLE
Can't ping outside from a container

### DISCUSSION THREAD

**Discussion by asifmuradkhan** [rancher-sandbox/rancher-desktop#2428] [Q&A]
Hi there,

I'm running rancher desktop on my intel mac, containers can ping each other but I'm not able to ping outside. Couldn’t find any setting to allow that, any idea?

thanks,

Asif

**Comment by mook-as** [comment]
Hi!

When you say you're unable to ping outside, what destination are you pinging? Somewhere on the (public) internet?

I ask because on mac (if you enter the `sudo` password), Rancher Desktop will create a network interface and obtain an IP address via DHCP on your local network; so if the "outside" is another local machine, it might have been blocked by a firewall.

Otherwise, there are issues with ICMP packets — `ping` might fail where a TCP connection (say, `curl www.google.com`) might succeed.

Hope that helps!
